templates/includes/sidebar.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block sidebar %}
+    <ul>
+        <li>
+            <a href="#">Home page</a>
+        </li>
+        <li>
+            <a href="#">Manufacturers</a>
+        </li>
+        <li>
+            <a href="#">Cars</a>
```

Comment:
And here as well, replace the hardcoded link with the `{% url 'url-name' %}` tag.